gn2/wqflask/templates/gnqa_search_history.html

```diff
@@ -1,42 +1,52 @@
-<section class="container-fluid  gnqa-copy">
+<section class="container-fluid  gnqa-copy" id="search-hist">
   <header class="row">
-
     <div class="panel panel default col-sm-6 col-sm-offset-3">
       <div class="panel panel-default">
 	<div class="panel-heading">
 	  <div>
-	    <h4 class="text-primary">You search History </h4>
+	    <h4 class="text-secondary" style="font-family: 'Linux Libertine','Georgia','Times','Source Serif Pro',serif;font-size:2.3rem">Your AI search History </h4>
 	  </div>
 	</div>
       </div>
     </div>
   </header>
   <div class="container row">
-  <div class="panel panel-default col-sm-6 col-sm-offset-3 ">
-    {% for record in prev_queries %}
-    <div class="panel-body">
-      <div class="row">
-	<input name="" type="checkbox" value="" class="col-sm-1">
+    <div>
+      <div class="col-sm-6 col-sm-offset-3" style="margin-bottom:10px">
+	<button type="button" class="btn btn-danger" id="delete-btn">Delete Selected </button>
+      </div>
+      <div >
+      <div class="panel panel-default col-sm-6 col-sm-offset-3 ">
+      <div>
+	<ul class="list-group list-group-flush" style="overflow-y:scroll">
+    {% for item in  prev_queries %}
+    <li class="row list-group-item">
+      	<input name="" type="checkbox"   value="{{item['task_id']}}" class="col-sm-1" style="height: 20px;
+  width: 20px;">
 	<div class="col-sm-10">
-	  {% for id,val in  record.items() %}
 	    <button
-	      hx-get="/gnqna/hist/search/{{id}}"
+	      hx-get="/gnqna/hist?query={{item['query']}}&search_term={{item['task_id']}}"
```

Comment:
I think jinja recommends having extra spaces after `{{` and before `}}` e.g. `{{ item['query'] }}`. Could we change all jinja refs to use this instead? [ref](https://jinja.palletsprojects.com/en/3.1.x/templates/)

Reply:
issue resolved by running djlint for gnqa files see commit:
https://github.com/genenetwork/genenetwork2/pull/863/commits/2c13a6224aa1fe7cd223a2c9cde237758ecfee7d